label_studio/ml_models/models.py

```diff
@@ -6,6 +6,15 @@
 from django.utils.translation import gettext_lazy as _
 from ml_model_providers.models import ModelProviderConnection
 from projects.models import Project
+from rest_framework.exceptions import ValidationError
+
+def validate_string_list(value):
+    if not value:
+        raise ValidationError('list should not be empty')
+    if not isinstance(value, list):
+        raise ValidationError('Value must be a list')
+    if not all(isinstance(item, str) for item in value):
```

Comment:
If we're validating that `output_classes` and `input_fields` are lists of strings, why do we store them as a `JSONField`? Is this just the closest type we can get to a list of strings?

Reply:
in django that is how list of string is being stored based what I have read.